ui/src/App.tsx

```diff
@@ -36,13 +36,13 @@ const authProvider = goOidcAgentAuthProvider(backend);
 const baseDataProvider = simpleRestProvider(
   `${backend}/api`,
   fetchJson,
-  "X-Total-Count"
+  "X-Total-Count",
```

Comment:
It looks like changes to the UI broke the ui-lint job

Reply:
I just ran the `make all -j`, and it generated this.. going to exclude it from the PR.. but not sure why what's getting generated is not right.